pkg/registry/core/backupentry/strategy.go

```diff
@@ -79,6 +86,21 @@ func mustIncreaseGeneration(oldBackupEntry, newBackupEntry *core.BackupEntry) bo
 		return true
 	}
 
+	if v1beta1helper.HasOperationAnnotation(newBackupEntry.Annotations) {
```

Comment:
We are now using the `GenerationChanged` predicate, so if we want to enqueue the backupEntry, we have to increase the generation, or else I'll have to `Or` the predicate with `HasOperationAnnotation` predicate. Can this be the reason for the flake? I changed this because, it was causing multiple reconciliations for restore case because of the annotation.